api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -1003,4 +1006,23 @@ public void applyLogLevels_shouldWarnWhenCalledWithInvalidLevel() {
 			context.updateLoggers();
 		}
 	}
+	
+	/**
+	 * @see OpenmrsUtil#conceptListHelper(String)
+	 */
+	@Test
+	public void conceptListHelper_shouldNotAllowDuplicateConcepts() {
+		ConceptService conceptService = Context.getConceptService();
+
+		String descriptor = "name:YES | name:NO | name:YES";
+		List<Concept> ret = OpenmrsUtil.conceptListHelper(descriptor);
+		assertEquals(2, ret.size());
+		
+		// both set point to the same concept with id 22		
+		descriptor = "set:30 | set:29";
```

Comment:
This as well... :)

Reply:
This is just a testing string and is specific to this particular test only and not utilized else where in other tests. Will it be fine to declare it as a global constant for all tests of this class? Can there be any reason to declare it as a global String ?